libs/wingsdk/src/target-tf-gcp/bucket.ts

```diff
@@ -32,6 +33,11 @@ const BUCKET_NAME_OPTS: NameOptions = {
   includeHash: false,
 };
 
+enum StorageBucketPermissions {
+  READ = "roles/storage.objectViewer",
+  READWRITE = "roles/storage.objectAdmin",
```

Comment:
https://cloud.google.com/iam/docs/understanding-roles#cloud-storage-roles

Isn't `objectAdmin` too much? I believe we should stick to the [Zero Trust](https://cloud.google.com/blog/topics/developers-practitioners/zero-trust-and-beyondcorp-google-cloud) model. I wonder if [objectUser](https://cloud.google.com/iam/docs/understanding-roles#cloud-storage-roles) would work here, though it doesn't have a note:
```
Lowest-level resources where you can grant this role:

Bucket
```
But it seems to have all the needed permissions. Can we apply it here?

Reply:
Yes, I also believe "roles/storage.objectUser" will be perfect fit in this scenario. Thanks for pointing this out.